packages/@glimmer/runtime/lib/dom/props.ts

```diff
@@ -46,14 +46,7 @@ export function normalizePropertyValue(value: Opaque): Opaque {
 // * strange spec outlier
 const ATTR_OVERRIDES = {
 
-  // phantomjs < 2.0 lets you set it as a prop but won't reflect it
-  // back to the attribute. button.getAttribute('type') === null
-  BUTTON: { type: true, form: true },
-
   INPUT: {
-    // Some version of IE (like IE9) actually throw an exception
-    // if you set input.type = 'something-unknown'
-    type: true,
```

Comment:
Can you confirm we have tests for this case (that would run under all of our supported browsers)?  I'm slightly concerned that this affected more browsers than just IE9...

Reply:
Added a test to make sure that the browser doesn't throw.

Reply:
❤️ thank you!